### PR TITLE
add chameleon

### PR DESCRIPTION
**Pull request by Armodillomatt12**
### Prerequisites
* [x] I have read and understood the [contribution guidelines](CONTRIBUTING.md)
* [x] This pull request follows the code style of the project
* [x] I have tested this feature thoroughly WITH THE USE of the load-all-sounds starter
  * [Link to a video of proof of testing](https://www.youtube.com/watch?v=VxsE56tpJ_o)

**Issues Fixed:** [Any issues fixed by this pull request]

### Changes Proposed in this Pull Request
* added cues for chameleon
* added chameleon to games.json

...

### CODE REVIEW

**Review by GuardedLolz — CHANGES_REQUESTED** [review]
The bug buzzing (not the prepare sounds) should loop.

**Review by GuardedLolz — CHANGES_REQUESTED** [review]
It's "Ja", not "Jp". Also, no folder for the Japanese Count-Ins.

**Comment by GuardedLolz** [comment]
Still, Ja, not Jp. Also, deprecations.

**Comment by Armodillomatt12** [comment]
i don't know how to change names or delete folders... i need help

**Comment by GuardedLolz** [comment]
And I mean a proper loop. Intro sound included. And stretchable.

File: core/assets/sounds/cues/countInJp/data.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
+{
```
Comment (chrislo27) [inline core/assets/sounds/cues/countInJp/data.json:1]: Remove the countInJp folder

File: core/assets/sounds/cues/countInEn/data.json [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        {
+          "id": "countInEn/four",
+          "beat": 7
+        }
+      ]
+    },
+    {
+      "id": "countInEn_4BeatCountInCowbell",
+      "deprecatedIDs": [
+        "countinen_4BeatCountInCowbell"
```
Comment (chrislo27) [inline core/assets/sounds/cues/countInEn/data.json:132]: You clearly didn't test this with deprecations. This is not a valid deprecated ID.

File: core/assets/sounds/cues/countInEn/data.json [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+  "cues": [
+    {
+      "id": "countInEn/silence",
+      "duration": 1,
+      "canAlterDuration": true,
+      "loops": false,
+      "name": "silence"
+    },
+    {
+      "id": "countInEn/cowbell",
```
Comment (chrislo27) [inline core/assets/sounds/cues/countInEn/data.json:13]: Why do none of these have deprecations?

**Review by chrislo27 — CHANGES_REQUESTED** [review]
Test things **thoroughly** next time.

File: core/assets/sounds/cues/countInJp/data.json (hunk 1 above)
Comment (Armodillomatt12) [reply core/assets/sounds/cues/countInJp/data.json:1]: i don't know how to

**Pull request closed without merging**